### ISSUE TITLE
Is there a difference between using in_out ratio compared to out_in if the value is 1?

### ISSUE THREAD

**Issue by manuelma** [spine-tools/SpineOpt.jl#640]

**Comment by jkiviluo** [comment]
I don't see any difference. And there wouldn't be a difference between in_out 2 and out_in 0.5 either, right? But is this an issue? I do find in_out/out_in confusing, but that's a separate discussion.

**Comment by manuelma** [comment]
I updated the description, sorry it wasn't clear at all...

**Comment by DillonJ** [comment]
Don't forget that the [min/max/fix]_ratio_[in/out]_[in/out] also have associated [min/max/fix]_ratio_[in/out]_[in/out]_units_on coefficient and in that case, it changes what side of the constraint the units_on term appears

**Comment by manuelma** [comment]
Good point @DillonJ I updated the title this time, so the suspicion is only for the 'fix' case.

**Comment by DillonJ** [comment]
@manuelma there is also a difference in the fix case as the units_on coefficient will effectively have a different sign depending on whether it is in/out or out/in

**Comment by DillonJ** [comment]
So to be more precise - the `fix_ratio_out_in_unit_flow` and `fix_ratio_in_out_unit_flow` are equivalent if they are both 1 _and_ `fix_units_on_coefficient_in_out` and `fix_units_on_coefficient_out_in` are both

**Comment by clizbe** [comment]
@manuelma Is this still an issue or did you figure it out?

**Comment by Tasqu** [comment]
This should no longer be an issue after #1276 finally gets merged.

It will still be possible to define two conflicting ratios, though, but at least they will no longer be completely identical in meaning.

**Comment by Tasqu** [comment]
Maybe less of an issue in #1276, although it's still possible to define two `flow_ratio_equality_coefficient`s for `uf1__uf2` and `uf2__uf1` separately. I suspect this will currently result in two separate constraints generated, thus potentially resulting in infeasibilities if different values are given for the different "directions" of the ratio.

I guess technically we could go through these in pre-processing to prevent conflicting definitions, or at the very least give some warnings. I don't see this as a priority, though, as I hope users are unlikely to give multiple ratio definitions unless they know what they're doing.
